manifests/p/PhilipGerke/Odassey/2.1.0/PhilipGerke.Odassey.locale.en-US.yaml

```diff
@@ -0,0 +1,27 @@
+# Created using YamlCreate.ps1 v1.1.7
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: PhilipGerke.Odassey
+PackageVersion: 2.1.0
+PackageLocale: en-US
+Publisher: Ingenieurbüro Philip Gerke
```

Comment:
```suggestion
Publisher: Philip Gerke
```

Reply:
`Publisher` must match Control Panel for WinGet to properly map this application to `winget list`.